README.md

```diff
@@ -23,7 +23,13 @@ In the following we define the message structure for the messages
 written to a network identity feed under the root meta feed.
 
 ```
-{ "type" => "metafeed/add", "feedpurpose" => "networkidentities", "subfeed" => (BFE feed ID) }
+{ 
+  "type" => "metafeed/add/derived", 
+  "feedpurpose" => "network-identities",
+  "subfeed" => (BFE-encoded Bendy Butt feed ID),
+  "metafeed" => (BFE-encoded Bendy Butt feed ID for the meta feed),
+  "nonce" => (bencode byte sequence with 32 random bytes),
+}
```

Comment:
I'm assuming this message is on the root meta feed (this spec doesn't actually say), and that's why I changed `metafeed/add` to `metafeed/add/derived`.

Note `networkidentities` => `network-identities`

Reply:
It should be right above: *network identity feed under the root meta feed*. Thanks the derived is correct.

Reply:
It just sounded like "following" meant this code block and that this was for "messages written to a network identity feed"

Reply:
I see, I updated the PR https://github.com/ssb-ngi-pointer/ssb-network-identity-spec/pull/2 to make that a bit clearer I hope. Thanks for the feedback :)